cg/store/models.py

```diff
@@ -1075,6 +1075,37 @@ class IlluminaSequencingRun(InstrumentRun):
     __mapper_args__ = {"polymorphic_identity": DeviceType.ILLUMINA}
 
 
+class PacBioSequencingRun(InstrumentRun):
+    __tablename__ = "pacbio_sequencing_run"
+
+    id: Mapped[int] = mapped_column(ForeignKey("instrument_run.id"), primary_key=True)
+    well: Mapped[Str32 | None]
+    plate: Mapped[int | None]
+    movie_time_hours: Mapped[int | None]
```

Comment:
`datetime.timedelta` is not supported by MySQL, we would have to do some extra work on the model to parse it as such, I think we should store the number of hours as ints. It can only be 12, 24 or 30 anyway